Backend/app/rider-platform/rider-app/Main/spec/Storage/FrfsTicket.yaml

```
@@ -34,6 +34,8 @@ imports:
   MultiModalJourneyRouteDetails: Lib.JourneyLeg.Types
   IntegratedBPPConfig: Domain.Types.IntegratedBPPConfig
   RecentLocation: Domain.Types.RecentLocation
+  ServiceTierType: BecknV2.FRFS.Enums
```

YAMLlint (1.35.1)

[error] duplication of key "ServiceTierType" in mapping

(key-duplicates)